.github/workflows/on_pull_request.yaml

```diff
@@ -24,13 +24,13 @@ jobs:
 
   tests:
     name: Run Tests
-    needs:
-      - lint
     uses: ./.github/workflows/tests.yaml
 
-  # publish runs in parallel with tests, as we always publish in this situation
   publish-charm:
     name: Publish Charm
+    needs:
```

Comment:
I think we agreed to
- require lint tests to pass before launching unit tests
- require unit tests to pass before integration
- require all above tests to pass before publishing on push
- but always publish the charm on pull request

So you don't need the tests to pass before publishing on pull request here. What do you think?

Reply:
You are right, I updated the flows.